src/System.Windows.Forms/src/System/Windows/Forms/RichTextBox.cs

```diff
@@ -3027,10 +3027,10 @@ private unsafe void SetCharFormatFont(bool selectionOnly, Font value)
                 FaceName = logFont.FaceName
             };
 
-            UnsafeNativeMethods.SendMessage(
-                new HandleRef(this, Handle),
-                RichEditMessages.EM_SETCHARFORMAT,
-                selectionOnly ? RichTextBoxConstants.SCF_SELECTION : RichTextBoxConstants.SCF_ALL,
+            User32.SendMessageW(
+                this,
+                (User32.WM)RichEditMessages.EM_SETCHARFORMAT,
+                selectionOnly ? (IntPtr)SCF.SELECTION :(IntPtr) SCF.ALL,
```

Comment:
Incorrect spacing. I propose we change it slightly:
```suggestion
                (IntPtr)(selectionOnly ? SCF.SELECTION : SCF.ALL),
```